cico_setup.sh

```diff
@@ -46,6 +46,7 @@ function build_planner() {
 function run_unit_tests() {
     # Run unit tests
     docker exec $CID npm run tests -- --unit
+    docker exec $CID "bash <(curl -s https://codecov.io/bash) -t 73933b5a-4aba-4b55-8612-a809ca4ada30"
```

Comment:
The token is a secret, right?
What about using `CODECOV_TOKEN` environment variable to set token?

Reply:
This isn't a secret actually. It's the `repo token`. This token is used to distinguish one repository from another.